docs/resources/run.md

```diff
@@ -40,7 +40,27 @@ resource "spacelift_run" "this" {
 - `commit_sha` (String) The commit SHA for which to trigger a run.
 - `keepers` (Map of String) Arbitrary map of values that, when changed, will trigger recreation of the resource.
 - `proposed` (Boolean) Whether the run is a proposed run. Defaults to `false`.
+- `timeouts` (Block, Optional) (see [below for nested schema](#nestedblock--timeouts))
+- `wait` (Block List, Max: 1) Wait for the run to finish (see [below for nested schema](#nestedblock--wait))
 
 ### Read-Only
 
 - `id` (String) The ID of the triggered run.
+
+<a id="nestedblock--timeouts"></a>
+### Nested Schema for `timeouts`
+
+Optional:
+
+- `create` (String)
+
+
+<a id="nestedblock--wait"></a>
+### Nested Schema for `wait`
+
+Optional:
+
+- `continue_on_discarded` (Boolean) Continue if run has been discarded. Default: `false`
```

Comment:
Would it make sense to create a single `[]string` field like `continue_on` where users can add any final state they want?

Reply:
@marcinwyszynski as requested, now using `continue_on_state` which accepts a list of possible (acceptable) end states. The `finished` state is the default state to wait for. I doubt Spacelift will change that name any soon.